images/cfw/etc/init.d/S70x1plus_sshd

```diff
@@ -6,6 +6,7 @@ HOST_KEYFILE="${CONFIG_DIR}/dropbear_ecdsa_host_key"
 PIDFILE="/var/run/x1plus_sshd.pid"
 
 update_rootpw() {
+  # TODO: Migrate to x1p_get_setting once bbl_screen moves over to settings.json
```

Comment:
Agree.  Or x1plus_sshd might be controlled by x1plusd, so it will react to settings changes.